doc/source/configuration/vault.rst

```diff
@@ -6,6 +6,18 @@ This document describes how to deploy Hashicorp Vault for
 internal PKI purposes using the
 `StackHPC Hashicorp collection <https://galaxy.ansible.com/stackhpc/hashicorp>`_
 
+Vault may be used as a Certificate Authority to generate certificates for:
+
+* OpenStack internal API
+* OpenStack backend APIs
+* RabbitMQ
+
+TLS support is described in the :kolla-ansible-doc:`Kolla Ansible documentation
```

Comment:
I can see that magic links like this work but how? Where is the `kolla-ansible-doc` function defined?

Reply:
See the extlinks stuff in doc/source/conf.py

Reply:
and https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html

Reply:
Ah, now I can see why I couldn't grep for it. Cool stuff